lib/es6-promise/promise.js

```diff
@@ -155,6 +155,9 @@ Promise.all = all;
 Promise.race = race;
 Promise.resolve = Resolve;
 Promise.reject = Reject;
+// _setScheduler() and _asap() must be non-enumerable
+Object.defineProperty(Promise, '_setScheduler', {value: setScheduler});
```

Comment:
es6-promise is supposed to work in es3 browsers still. Unfortunately, no define prop :(